src/event-replay/event-replay.ts

```diff
@@ -138,14 +144,12 @@ export async function importEventsFromTsv(
 
   // Import TSV chain data
   const readStream = fs.createReadStream(resolvedFilePath);
-  const rawEventsIterator = PgDataStore.getRawEventRequests(readStream, status => {
+  const rawEventsIterator = getRawEventRequests(readStream, status => {
     console.log(status);
   });
   // Set logger to only output for warnings/errors, otherwise the event replay will result
   // in the equivalent of months/years of API log output.
   logger.level = 'warn';
-  // Disable this feature so a redundant export file isn't created while importing from an existing one.
-  delete process.env['STACKS_EXPORT_EVENTS_FILE'];
```

Comment:
Is this assuming the `STACKS_EXPORT_EVENTS_FILE` ENV var won't be present in the environment when running an event replay? What happens if it still is? Would it cause problems?

Reply:
No worries... since it's not used anywhere in code, having it defined in ENV will not cause any side effects